steps/s3/session.go

```diff
@@ -48,16 +49,27 @@ func (s *Session) NewS3Session(ctx context.Context) error {
 	logger := GetLogger()
 	logger.LogMessage("Creating a new S3 session")
 
+	// Set aws region
+	var region string
+	if golium.GetEnvironment().Get("s3Region") == nil {
+		if region = os.Getenv("AWS_REGION"); len(region) == 0 {
+			return fmt.Errorf("error s3 region must be set via s3Region config parameter or AWS_REGION environment var")
+		}
+	} else {
+		region = golium.Value(ctx, "[CONF:s3Region]").(string)
```

Comment:
So that, s3Region should be enabled as env variable and also as conf variable to work?

Reply:
What this approach tries to implement is: If CONF:s3Region is set use it, If not look for AWS_REGION.  
In the case that no region is available do not allow the creation of a S3 session.